### ISSUE TITLE
Dropout not implemented in FCs

### ISSUE THREAD

**Issue by hchintada** [ArkinDharawat/DeepImageRanking#3]
The paper says that a dropout layer with prob rate of 0.6 is implemented for all the fully connected layers in the network.
Curious to know the reason why it is not added in the architecture given in the repo!

Do you suggest adding a dropout layer?

**Comment by ArkinDharawat** [comment]
Yes, feel free to add a dropout layer and create a pull request.
